### DISCUSSION TITLE
Tracking external queries from DBeaver

### DISCUSSION THREAD

**Discussion by rigczTheSky** [dbeaver/dbeaver#22500] [Q&A]
Hi. I'm working on application connected to PostgreSQL database. I am wandering is it possible to see sql queries executed from this application by DBeaver or maybe you know other tools to monitor PostgreSQl activity?

**Comment by LonwoLonwo** [comment]
Hello @rigczTheSky 

Maybe the [Session Manager](https://github.com/dbeaver/dbeaver/wiki/Session-Manager-Guide) will be helpfull in your case?
